core/src/main/java/org/dcache/nfs/v3/MountServer.java

```diff
@@ -83,8 +83,7 @@ public mountres3 MOUNTPROC3_MNT_3(RpcCall call$, dirpath arg1) {
 
         mountres3 m = new mountres3();
 
-        java.io.File f = new java.io.File(arg1.value);
-        String mountPoint = f.getAbsolutePath();
+        String mountPoint = arg1.value;
```

Comment:
If I recall correctly, the reason of converting to File and then getting absolute path is to get rid of redundant slashes ('/') in argument to mount request.

Looks like Paths.get() method does what you need:

https://docs.oracle.com/javase/tutorial/essential/io/pathOps.html

Can you give it a try?

Reply:
Paths.get will create a relative path, yes. Still have to keep the slashes converted because the slashes will be backwards ('\', or '\\' more specifically for the literal) which would fail in the comparison in the getExport.  Latest commit has that change.